### ISSUE TITLE
[accessibility] TOGGLE_MAXIMIZED overrides monaco tab capture

### ISSUE THREAD

**Issue by caseyflynn-google** [eclipse-theia/theia#6996]
### Description
 

The TOGGLE_MAXIMIZE keybinding overrides Monaco's toggleTabFocusMode which is required for a11y. Would it be alright if we modify the keybinding for TOGGLE_MAXIMIZE (maybe ctrl+shift+m or alt+m)?

### Reproduction Steps

1. Open chrome developer tools.
2. Load Theia

Observe warning in console:
root WARN Collided keybinding is ignored;  {"command":"editor.action.toggleTabFocusMode","keybinding":"ctrl+m"}  collided with  {"command":"core.toggleMaximized","keybinding":"ctrl+m"}

**OS and Theia version:**

ChromeOs
Theia 0.14.0

**Comment by akosyakov** [comment]
yes, it should be fine
